### PR TITLE
Dashboard/review news

### PR DESCRIPTION
**Pull request by heyitsmaimai**

### CODE REVIEW

File: src/dashboard/cd-dashboard-news.vue [hunk 1]
```diff
@@ -164,14 +141,11 @@
       }
 
       &-title {
-        margin: 0;
-
-        &-link {
+        font-weight: bold;
```
Comment (Wardormeur) [inline src/dashboard/cd-dashboard-news.vue]: Wont' that look very... aggressive ? I mean, there is 6 news displayed next to another.
Could we get a screenshot ? :)

File: src/dashboard/cd-dashboard-news.vue [hunk 2]
```diff
@@ -114,36 +112,15 @@
...
-        flex-direction: column;
-        margin: 0 16px 0 16px;
-        max-width: 30%;
+        max-width: 75px;
       }
 
       &-right{
         margin: 0 16px 0 16px;
-        max-width: 70%;
+        max-width: 425px;
```
Comment (Wardormeur) [inline src/dashboard/cd-dashboard-news.vue]: What happens when it's over 425 (ie responsive)?

File: src/dashboard/cd-dashboard-news.vue (hunk 1 above)
Comment (heyitsmaimai) [reply src/dashboard/cd-dashboard-news.vue]: i'll put one up on the card in a minute

File: src/dashboard/cd-dashboard-news.vue (hunk 2 above)
Comment (heyitsmaimai) [reply src/dashboard/cd-dashboard-news.vue]: it wraps. but also need to fix mobile padding for the date

Comment (Wardormeur) [reply src/dashboard/cd-dashboard-news.vue]: Fix it and it's a go :)

Comment (heyitsmaimai) [reply src/dashboard/cd-dashboard-news.vue]: it's fixed!!!

**Comment by Wardormeur** [comment]
I believe in you :<

**Pull request merged**